Unboxing

## `installation experiences`


### initial setup
```
# get source
git clone git@github.com:bmeg/bmeg-proxy.git
# correct branch
git checkout dev
# npm - all the JS 
npm i -S
# install hugo - static site generator
brew install hugo

```

### hickup
```
$ make server
./proxy.py http://bmeg.io
Traceback (most recent call last):
  File "./proxy.py", line 5, in <module>
    import tornado
ImportError: No module named tornado
make: *** [server] Error 1
```
#### fix
```
# add to README
pip install -r requirements.txt
```

### continue install

```
make
make server
```

--- a/README.md
+++ b/README.md
@@ -18,6 +18,9 @@ or
 
   apt-get install hugo
   
+# Install python dependencies
+
+  pip install -r requirements.txt
 
 To Build the site:
 

